src/Rules/Install.hs

```diff
@@ -193,6 +193,20 @@ installPackages = do
                 let ghcCabalInplace = inplaceBinPath -/- "ghc-cabal" -- HACK?
                 need [ ghcCabalInplace ]
 
+                let cabalFile = pkgCabalFile pkg
+                -- HsColour sources
+                -- QUESTION: what is the output of GhcCabalHsColour?
+                whenM (isSpecified HsColour) $ do
+                    top <- interpretInContext context getTopDirectory
+                    let installDistDir = top -/- buildPath context
+                    -- HACK: copy stuff back to the place favored by ghc-cabal
+                    quietly $ copyDirectoryContents (Not excluded)
```

Comment:
`GhcCabalHsColour` will invoke `ghc-cabal`, which assumes the path (`build`). So this is basically the same problem as #318.

So, copying has to be done before `GhcCabalHsColour`, or `installPackages` (where we use `ghc-cabal copy`.